tools/plotheatmap/plotheatmap.xml

```diff
@@ -0,0 +1,87 @@
+<tool id="plotheatmap" name="plotHeatmap" version="1.0">
+    <description>Plot heatmap</description>
+    <requirements>
```

Comment:
The Travis test is failing because you're missing a requirement here:
```xml
        <requirement type="package" version="1.34.0">bioconductor-preprocesscore</requirement>
```